drltrace/drltrace_frontend.cpp

```diff
@@ -302,11 +308,25 @@ _tmain(int argc, const TCHAR *targv[])
         tmp--;
     *(tmp+1) = '\0';
 
+    /* in case of default config option, we use drltrace's install path */
+    if (op_config_file.get_value().compare(DEFAULT_CONFIG_DIR) == 0) {
```

Comment:
It looks like my comments here and below are wrong. Actually I'm setting drltrace's binary path (not install path) here, fixed comments. I've added a new boolean option similarly to the Dr.Memory to disable/enable default config file usage.